aws-proofs/steps.sh

```diff
@@ -44,30 +44,33 @@ until nc -w5 -z ${IP} 22; do echo "."; sleep 3; done
 eval $(ssh-agent)
 ssh-add -q - <<< "${AWS_SSH}"
 
+# Prime GH runner with VM ssh host key. This does not gain any actual security,
+# because we have nothing to verify the key against. It only make ssh/scp less chatty.
+mkdir -p ~/.ssh
+ssh-keyscan -t ecdsa ${IP} 2> /dev/null >> ~/.ssh/known_hosts
+
 if [ "${GITHUB_EVENT_NAME}" = "pull_request_target" ]
 then
-  GH_REF=${GH_HEAD_SHA}
-else
-  GH_REF=${GITHUB_REF}
+  GITHUB_REF=${GH_HEAD_SHA}
 fi
 
 echo "::endgroup::"
 
-CI_BRANCH=master
-
-ssh -o StrictHostKeyChecking=no test-runner@${IP} \
-    "bash -c \"export INPUT_L4V_ARCH=${INPUT_L4V_ARCH}; \
-               export INPUT_MANIFEST=${INPUT_MANIFEST}; \
-               export INPUT_ISA_BRANCH=${INPUT_ISA_BRANCH}; \
-               export INPUT_SESSION=${INPUT_SESSION}; \
-               export INPUT_CACHE_NAME=${INPUT_CACHE_NAME}; \
-               export INPUT_CACHE_READ=${INPUT_CACHE_READ}; \
-               export INPUT_CACHE_WRITE=${INPUT_CACHE_WRITE}; \
-               export GITHUB_REPOSITORY=${GITHUB_REPOSITORY}; \
-               export GITHUB_REF=${GH_REF}; \
-               export GITHUB_BASE_REF=${GITHUH_BASE_REF}; \
-               export GITHUB_WORKSPACE=/home/test-runner; \
-               ./run ${CI_BRANCH}\""
+INPUT_CI_BRANCH=master
```

Comment:
This is `INPUT_CI_BRANCH` now because the VM is configured to only accepts `INPUT_*` and `GITHUB*` environment variables.